go/pkg/coordinator/grpc/server.go

```diff
@@ -44,13 +44,20 @@ type Config struct {
 	PulsarNamespace string
 
 	// Kubernetes config
-	KubernetesNamespace  string
-	WorkerMemberlistName string
-	WorkerPodLabel       string
+	KubernetesNamespace string
+
+	// Query service memberlist config
+	QueryServiceMemberlistName string
+	QueryServicePodLabel       string
 
 	// Watcher config
 	WatchInterval time.Duration
 
+	// Compaction service memberlist config
+	// TODO: We need to refactor memberlist manager to support multiple memberlists
```

Comment:
personally think how it is is perfectly fine given we are N=2 for the indefinite future.